traveler-ios-ui/TravelerKitUI/ModelExtensions/PartnerOffering+TravelerUI.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  PartnerOffering+TravelerUI.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-12-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import Foundation
+import TravelerKit
+
+extension PartnerOffering {
```

Comment:
Suggestion: If we don't want to extend NSMutableAttributedString, we should have a common util with static functions that we can call to do this instead.  i.e. GLXStringUtil.htmlAttributedString(...).  We keep rewriting the same thing over and over again.

Reply:
Added todo comment, I'm having @benRmc document all our TODOs into a confluence page so that we can revisit them after xmas.